src/tokens/spacing.json

```diff
@@ -0,0 +1,36 @@
+{
+  "spacing": {
+    "base": {
+      "value": ".5rem",
+      "comment": "8px"
+    },
+    "xxs": {
+      "value": "({spacing.base.value} / 2)",
```

Comment:
Let's go ahead and do this math and use the resulting values here that way the output in various formats will be usable without having to manipulate a to do math in the target language. Seems like you can still do math in the source tokens, but it will need moved inside the the curly brackets, which I'm assuming mark the start of a JavaScript expression where you could then do the math. I'm not sure about how that works in style dictionary, so you'll need to confirm 👍 

```js
{spacing.basevalue / 2}
```